corehq/toggles.py

```diff
@@ -282,6 +282,12 @@ def toggles_dict(username=None, domain=None):
     [NAMESPACE_DOMAIN, NAMESPACE_USER]
 )
 
+LOCATION_TYPE_STOCK_RATES = StaticToggle(
+    'location_type_stock_rates',
+    "Specify stock rates per location type.",
```

Comment:
Yeah, we basically don't follow it at all, but I generally try to in my code.  I think it's a pretty weakly adhered-to convention anyways.